frontend/package.json

```diff
@@ -74,7 +74,7 @@
     "deploy-azure-training": "REACT_APP_OKTA_URL=https://training.simplereport.gov REACT_APP_BACKEND_URL=https://training.simplereport.gov/api REACT_APP_IS_TRAINING_SITE=true PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttrainingapp --destination-path '/app'",
     "deploy-azure-demo": "REACT_APP_OKTA_URL=https://demo.simplereport.gov REACT_APP_BACKEND_URL=https://demo.simplereport.gov/api REACT_APP_BASE_URL=https://demo.simplereport.gov REACT_APP_PATIENT_EXPERIENCE_ENABLED=true PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdemoapp --destination-path '/app'",
     "deploy-azure-stg": "REACT_APP_OKTA_URL=https://stg.simplereport.gov REACT_APP_BACKEND_URL=https://stg.simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportstgapp --destination-path '/app'",
-    "deploy-azure-prod": "REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b REACT_APP_OKTA_URL=https://simplereport.gov REACT_APP_BACKEND_URL=https://simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'",
+    "deploy-azure-prod": "REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b REACT_APP_OKTA_URL=https://simplereport.gov REACT_APP_BACKEND_URL=https://simplereport.gov/api PUBLIC_URL=/app/ REACT_APP_PATIENT_EXPERIENCE_ENABLED=true yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'",
```

Comment:
Can we just flip this on in the frontend `.env` file? That what we've been doing with the other flags I think.

Reply:
updated to flip the flag on for every env.

Reply:
This seems like a situation that calls for an actual config file, but I have the feeling that configuration management for multiple deployed environments is a rabbit hole that yarn has stored some traps in.